replicate-to-sql-with-generic-events/scripts/utils.js

```diff
@@ -0,0 +1,28 @@
+async function createIndexIfNotExists(kuzzle, index = '') {
```

Comment:
Why a default parameter to `''` ? If the index is a blank string, the SDK method will fail